### ISSUE TITLE
Base64 Function - Insomnia has become unresponsive

### ISSUE THREAD

**Issue by vbrich** [getinsomnia/insomnia#2667]
**Describe the bug**
We have many scripts that contain xml data we are base64 encoding within Insomnia. The files are large, but not ridiculous (60kb). 

**To Reproduce**
1. Create a POST to any endpoint
2. Add a Base64=>Encode function
3. Paste a larger file in there
4. Do anything, e.g. try to cut/paste it out, try to put a new file in, etc

**Expected behavior**
Normally, the live preview in the "Edit Tag" will take a few seconds to render, then you can click Done

**Screenshots**
[
![Capture](https://user-images.githubusercontent.com/43274908/94301196-04ee0c00-ff1f-11ea-8228-fa46903aeb85.PNG)
](url)

**Desktop (please complete the following information):**
 - OS: Windows_NT x64 10.0.17763 (also happens on my MacOS)
 - Installation Method: Windows installer
 - App Version: 2020.4.0

**Additional context**
I believe this is tied to the release that happened on 9/9/2020. Many of us share these scripts, including partners. I'm hearing the exact same issue from all parties. All of our existing scripts now hang when we try to change data in our scripts. It only hangs when changing data in our Base64 Encode function.

**Comment by develohpanda** [comment]
Thank you for reporting this! This

**Comment by develohpanda** [comment]
Thank you for reporting this! Curiously, you mention this is happening with _larger_ files. Looking at the size of the scrollbar in the live preview, I think this is a performance limitation with `textarea`. See a related issue and fix in #2590.

I am fairly certain this isn't new to 2020.4.0, though (I don't see any related changes). Would you be able to confirm if you see the same behavior with the same file in the previous stable release, [2020.3.3](https://github.com/kong/insomnia/releases/tag/core%402020.3.3)?

**Comment by vbrich** [comment]
Great response time! Was expecting this to sit for a bit. 

I have uninstalled Insomnia, then re-installed 2020.3.3 (and turned off auto updates). Everything is working now. I will ask a couple of other people who were also seeing the issue to confirm this is what they see as well. 

Maybe it isn't "large files", but something within the actual data being base64 encoded? I can't post my sample workspace since the payload is proprietary, but I will try to recreate this with another dummy/test file. I'm sure a single script showing this in action would go a long way.

**Comment by develohpanda** [comment]
> I'm sure a single script showing this in action would go a long way.

Absolutely, that would be very useful to help reproduce it. I created an example value that gave me an encoded length similar to yours, and it seemed speedy - on v2020.4.1.
